### PR TITLE
Issue #12134 Solved in numbers.py and extra space added for symmetry of code

### PR DESCRIPTION
**Pull request by meet7202**
<!-- If this pull request fixes an issue please indicate which issue by typing "Fixes #NNNN" below.>
Issue #12134 Solved in numbers.py and extra space added for symmetry of code
--------------------------------------------------------------------
>>> import sympy
>>> x = sympy.numbers.Float(5)
>>> import numbers
>>> isinstance(x, numbers.Number)
True
------------------------------------------------------------------
In above code if you want to print x it will give you 
Decimal('5.00000000000000')
output in place of 5.00000000000000 but it will work fine with numbers as Decimal('5.00000000000000') is also a number.

### CODE REVIEW

**Comment by asmeurer** [comment]
No, this is not the right fix. You should change the superclass of Float. The decimal module should not come into play.

File: sympy/solvers/solvers.py [hunk 1]
```diff
@@ -350,6 +350,7 @@ def check_assumptions(expr, against=None, **assumptions):
         if test is expected:
             continue
         elif test is not None:
+            print("Assumption for (",expr,") is wrong as",key,"returned ",end="")
```
Comment (gxyd) [inline sympy/solvers/solvers.py]: Don't put `print` statements in your commits. Remove these.

**Comment by gxyd** [comment]
@meet7202 can you please merge master branch? and also please address the requested change.

**Comment by gxyd** [comment]
This PR has been inactive for quite sometime, so we are going to close it and open an issue to reference it. We try to keep our pull request queue small and focused on active work.  We encourage you to reopen the pull request if and when you continue to work on this. Please contact us if you have any questions.

Thanks for contributing.

see https://github.com/sympy/sympy/wiki/Closing-pull-requests for our policies on closing pull request.

The issue is already there for it https://github.com/sympy/sympy/issues/12134

**Pull request closed without merging**